R/run-nmtran.R

```diff
@@ -0,0 +1,223 @@
+#' Interface for running `NM-TRAN` on model objects
+#'
+#' Provides functions to run `NM-TRAN` on model objects for validation and to
+#' generate `NM-TRAN` datasets (`FDATA`).
```

Comment:
The last part has a leftover mention of FDATA.

Reply:
I noticed this too, but was thinking about keeping it since `FDATA` can technically still be viewed/read-in if you keep the folder around via `delete_on_exit = FALSE`. Do you think it's confusing as-is?

Reply:
I see.  I don't feel to strongly, but fwiw I do find it confusing.  I don't expect the user would make that connection, and it seems odd to highlight just one of the things NM-TRAN produces.  (Also, as I mentioned elsewhere, I'm not convinced we should expose `delete_on_exit`.)

Reply:
I adjusted the docs (mainly the description) to clarify here. Let me know if it's still confusing.